code/modules/mob/living/carbon/xenomorph/castes/king/abilities_king.dm

```diff
@@ -32,104 +32,272 @@
 
 
 // ***************************************
-// *********** Gravity Crush
+// *********** Petrify
 // ***************************************
-#define WINDUP_GRAV 1 SECONDS
-
-/datum/action/xeno_action/activable/gravity_crush
-	name = "Gravity Crush"
-	action_icon_state = "fortify"
-	mechanics_text = "Increases the localized gravity in an area and crushes everything in it."
-	ability_name = "Gravity crush"
-	plasma_cost = 200
+#define PETRIFY_RANGE 5
+#define PETRIFY_DURATION 5 SECONDS
+#define PETRIFY_WINDUP_TIME 2.5 SECONDS
+/datum/action/xeno_action/petrify
+	name = "Petrify"
+	action_icon_state = "petrify"
+	mechanics_text = "After a windup, petrifies all humans looking at you. While petrified humans are immune to damage, but also can't attack."
+	ability_name = "petrify"
+	plasma_cost = 100
 	cooldown_timer = 30 SECONDS
+	keybind_flags = XACT_KEYBIND_USE_ABILITY
+	keybinding_signals = list(
+		KEYBINDING_NORMAL = COMSIG_XENOABILITY_PETRIFY,
+	)
+
+/datum/action/xeno_action/petrify/can_use_action(silent, override_flags)
+	. = ..()
+	if(!.)
+		return
+	if(LAZYACCESS(owner.do_actions, src))
+		owner.balloon_alert(owner, "already busy")
+		return FALSE
+
+/datum/action/xeno_action/petrify/action_activate()
+	var/obj/effect/overlay/eye/eye = new
+	owner.vis_contents += eye
+	flick("eye_opening", eye)
+	playsound(owner, 'sound/effects/petrify_charge.ogg', 50)
+	if(!do_after(owner, PETRIFY_WINDUP_TIME, FALSE, owner, BUSY_ICON_DANGER))
+		flick("eye_closing", eye)
+		addtimer(CALLBACK(src, .proc/remove_eye, eye), 7, TIMER_CLIENT_TIME)
+		return
+	playsound(owner, 'sound/effects/petrify_activate.ogg', 50)
+	var/list/mob/living/carbon/human/humans = list()
+	for(var/mob/living/carbon/human/human in view(PETRIFY_RANGE, owner))
+		if(human.stat != CONSCIOUS)
+			continue
+		if(is_blind(human))
+			continue
+		humans += human
+		human.notransform = TRUE
```

Comment:
I think this entirely stops movements, but not sure about getting thrown or anything like that, would could lead to FUN.